configurationDC.py

```diff
@@ -0,0 +1,232 @@
+import os
+import logging
+from theme import THEME_DEFAULT
+
+class ConfigurationError(Exception):
+    """
+    Exception class for errors reading and writing configuration
+    """
+
+class ConfigurationWarning(Exception):
+    """
+    Warning class for errors reading configuration
+    """
+
+class WhisperAttackConfiguration:
```

Comment:
This should be named something different, i.e. shouldn't have two classes in different files with the same name. Maybe change the filename to the snakecase naming convention, e.g. `distributed_configuration.py` and class `WhisperAttackDistributedConfiguration`

Alternatively, are two different classes needed. Can two extra remote server config items just be added to existing one.